src/js/clipboard.js

```diff
@@ -0,0 +1,640 @@
+/*!
+ * clipboard.js v2.0.0
+ * https://zenorocha.github.io/clipboard.js
+ *
+ * Licensed MIT © Zeno Rocha
+ */
+!(function (t, e) {
+  "object" == typeof exports && "object" == typeof module
+    ? (module.exports = e())
+    : "function" == typeof define && define.amd
+    ? define([], e)
+    : "object" == typeof exports
+    ? (exports.ClipboardJS = e())
+    : (t.ClipboardJS = e());
+})(this, function () {
+  return (function (t) {
+    function e(o) {
+      if (n[o]) return n[o].exports;
+      var r = (n[o] = { i: o, l: !1, exports: {} });
+      return t[o].call(r.exports, r, r.exports, e), (r.l = !0), r.exports;
+    }
+    var n = {};
+    return (
+      (e.m = t),
+      (e.c = n),
+      (e.i = function (t) {
+        return t;
+      }),
+      (e.d = function (t, n, o) {
+        e.o(t, n) ||
+          Object.defineProperty(t, n, {
+            configurable: !1,
+            enumerable: !0,
+            get: o,
+          });
+      }),
+      (e.n = function (t) {
+        var n =
+          t && t.__esModule
+            ? function () {
+                return t.default;
+              }
+            : function () {
+                return t;
+              };
+        return e.d(n, "a", n), n;
+      }),
+      (e.o = function (t, e) {
+        return Object.prototype.hasOwnProperty.call(t, e);
+      }),
+      (e.p = ""),
+      e((e.s = 3))
+    );
+  })([
+    function (t, e, n) {
+      var o, r, i;
+      !(function (a, c) {
+        (r = [t, n(7)]),
+          (o = c),
+          void 0 !== (i = "function" == typeof o ? o.apply(e, r) : o) &&
+            (t.exports = i);
+      })(0, function (t, e) {
+        "use strict";
+        function n(t, e) {
+          if (!(t instanceof e))
+            throw new TypeError("Cannot call a class as a function");
+        }
+        var o = (function (t) {
+            return t && t.__esModule ? t : { default: t };
+          })(e),
+          r =
+            "function" == typeof Symbol && "symbol" == typeof Symbol.iterator
+              ? function (t) {
+                  return typeof t;
+                }
+              : function (t) {
+                  return t &&
+                    "function" == typeof Symbol &&
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)